administrators.rst

```diff
@@ -267,12 +264,21 @@ Transfer Source. This is the only type of location that will be allowed inside a
 Dataverse space.
 
 Within this location, the relative path can be used to store a query string,
-which is used to interrogate the Dataverse Search API (for example, q=* will
-return all datasets).
+which is used to interrogate the Dataverse Search API. The q parameter is a 
+general search parameter. The ’subtree’ parameter can be used to indicate a 
+sub-dataverse. For example:
+
+::
+
+   q=*
+   subtree=“Archivematica”
+
+will return all datasets contained within the “Archivematica” sub-dataverse. 
 
 The Storage Service will always add ``type=dataset`` to the calls that it makes
-to the Dataverse Search API. The automation tools will create one transfer for
-every dataset, which subsequently becomes one SIP and one AIP.
+to the Dataverse Search API. 
```

Comment:
I think this is good, you could elaborate further and note the defaults are constructed with the following parameters:
```
URL: https://demodv.scholarsportal.info/api/search/

{
	'q': '*',
	'sort': 'name',
	'key': u 'f247db44-8c06-42ef-a75b-eb5ffdc6e51b',
	'start': 50,
	'per_page': 50,
	'show_entity_ids': True,
	'type': 'dataset',
	'subtree': None,
	'order': 'asc'
}
```
So a user is invited to seek more specific searches where they want to access beyond the first 50 results.

Reply:
Thanks Ross, I think that is a good suggestion - I added this, but with some additional clarification of how each parameter gets set. It adds a certain amount of detail, but I think you are right it is important to point out that only 50 datasets are returned, and users should create more specific searches with multiple locations if they need more than that.

Reply:
I also tried to improve on the text you mention in your first comment... I agree it was a little awkward.